python/docs/source/examples/time_centric.ipynb

```diff
@@ -0,0 +1,339 @@
+    "* the total revenue from the user today\n",
+    "* the number of events the user has generated in the past hour\n",
+    "\n",
+    "Because the calculations needed here are a mix of hourly, daily, and over all of\n",
+    "history, more than one type of event aggregation needs to happen. Table-centric\n",
+    "tools like those based on SQL would require multiple JOINs and window functions,\n",
+    "which would be spread over multiple queries or CTEs. \n",
+    "\n",
+    "Kaskada was designed for these types of time-centric calculations, so we can do\n",
+    "each of the calculations in the list in one line:\n",
+    "\n",
+    "```python\n",
+    "record({\n",
+    "    \"event_count_total\": DemoEvents.count(),\n",
+    "    \"purchases_total_count\": DemoEvents.filter(DemoEvents.col(\"event_name\").eq(\"purchase\")).count(),\n",
+    "    \"revenue_total\": DemoEvents.col(\"revenue\").sum(),\n",
+    "    \"purchases_daily\": DemoEvents.filter(DemoEvents.col(\"event_name\").eq(\"purchase\")).count(window=Daily()),\n",
+    "    \"revenue_daily\": DemoEvents.col(\"revenue\").sum(window=Daily()),\n",
+    "    \"event_count_hourly\": DemoEvents.count(window=Hourly()),\n",
+    "})\n",
+    "```\n",
+    "\n",
+    "```{warning}\n",
+    "The previous example demonstrates the use of `Daily()` and `Hourly()` windowing which aren't yet part of the new Python library.\n",
+    "```\n",
+    "\n",
+    "Of course, a few more lines of code are needed to put these calculations to work,\n",
+    "but these six lines are all that is needed to specify the calculations\n",
+    "themselves. Each line may specify:\n",
+    "* the name of a calculation (e.g. `event_count_total`)\n",
+    "* the input data to start with (e.g. `DemoEvents`)\n",
+    "* selecting event fields (e.g. `DemoEvents.col(\"revenue\")`)\n",
+    "* function calls (e.g. `count()`)\n",
+    "* event filtering (e.g. `filter(DemoEvents.col(\"event_name\").eq(\"purchase\"))`)\n",
+    "* time windows to calculate over (e.g. `window=Daily()`)\n",
+    "\n",
+    "...with consecutive steps chained together in a familiar way.\n",
+    "\n",
+    "Because Kaskada was built for time-centric calculations on event-based data, a\n",
+    "calculation we might describe as \"total number of purchase events for the user\"\n",
+    "can be defined in Kaskada in roughly the same number of terms as the verbal\n",
+    "description itself.\n",
+    "\n",
+    "Continue through the demo below to find out how it works.\n",
+    "\n",
+    "See [the Kaskada documentation](../guide/index) for lots more information."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "BJ2EE9mSGtGB",
+   "metadata": {
+    "id": "BJ2EE9mSGtGB"
+   },
+   "source": [
+    "## Kaskada Client Setup\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "37db47ba",
+   "metadata": {
+    "tags": [
+     "hide-output"
```

Comment:
This causes the output to not show up (it's hidden / collapsed). We can also mark it as `remove-output` which would cause it not be in the page at all.